src/experiments.js

```diff
@@ -73,7 +76,27 @@ export function isDevChannelVersionDoNotUse_(win) {
  * @return {boolean}
  */
 export function isExperimentOn(win, experimentId) {
-  return getExperimentIds(win).indexOf(experimentId) != -1;
+  if (experimentId in EXPERIMENT_TOGGLES) {
+    return EXPERIMENT_TOGGLES[experimentId];
+  }
+
+  const cookieFlag = getExperimentIds(win).indexOf(experimentId) != -1;
+  if (cookieFlag) {
+    return EXPERIMENT_TOGGLES[experimentId] = cookieFlag;
+  }
+
+  if (win.AMP_CONFIG && win.AMP_CONFIG.hasOwnProperty(experimentId)) {
+    const expChance = win.AMP_CONFIG[experimentId];
+    if (expChance <= 0) {
```

Comment:
These conditions shouldn't be necessary. `expChance <= 0  === Math.random() < expChance` and `expChance >= 1 === Math.random() < expChance`